benchmark_runner/benchmark_operator/benchmark_operator_workloads.py

```diff
@@ -365,16 +391,24 @@ def uperf_pod(self):
             self.__oc.create_pod_sync(yaml=os.path.join(f'{self.__current_run_path}', f'{self.uperf_pod.__name__}.yaml'), pod_name=f'uperf-server')
             # uperf server
             server_name = self.__environment_variables_dict.get('pin_node1', '')
-            # name up to 27 chars
-            if len(server_name) > 27:
-                server_name = server_name[:28]
-            self.__oc.wait_for_initialized(label=f'app=uperf-bench-server-{server_name}-0', workload=workload)
-            self.__oc.wait_for_ready(label=f'app=uperf-bench-server-{server_name}-0', workload=workload)
+            if server_name:
+                # name up to 27 chars
```

Comment:
Why the fixed upper limit?

Reply:
When pin_node is missing we got empty value. so I will skip it when missing pin node

Reply:
That's not my question; why is there an uppoer limit of 27 characters?  The DNS limit is something like 250, as I recall.

Reply:
That is the limitation of Uperf workload in benchmark-wrapper, so I have set it accordingly.

Reply:
Then rather than just "name up to 27 chars", document it as "uperf workload is limited to 27 characters".  The "why" is more important than the "what".

Reply:
I dont have time to dig into it and understand why. I am focus on verifying that Uperf initialized properly